eclair-core/src/main/scala/fr/acinq/eclair/channel/ChannelData.scala

```diff
@@ -265,7 +265,7 @@ final case class RES_BUMP_FUNDING_FEE(rbfIndex: Int, fundingTxId: ByteVector32,
 final case class RES_SPLICE(fundingTxIndex: Long, fundingTxId: ByteVector32, capacity: Satoshi, balance: MilliSatoshi) extends CommandSuccess[CMD_SPLICE]
 final case class RES_GET_CHANNEL_STATE(state: ChannelState) extends CommandSuccess[CMD_GET_CHANNEL_STATE]
 final case class RES_GET_CHANNEL_DATA[+D <: ChannelData](data: D) extends CommandSuccess[CMD_GET_CHANNEL_DATA]
-final case class RES_GET_CHANNEL_INFO(nodeId: PublicKey, channelId: ByteVector32, state: ChannelState, data: ChannelData) extends CommandSuccess[CMD_GET_CHANNEL_INFO]
+final case class RES_GET_CHANNEL_INFO(requestId: ByteVector32, nodeId: PublicKey, channelId: ByteVector32, state: ChannelState, data: ChannelData) extends CommandSuccess[CMD_GET_CHANNEL_INFO]
```

Comment:
This is the approach we are using on a feature branch, to the same effect:

```suggestion
final case class RES_GET_CHANNEL_INFO(nodeId: PublicKey, channelId: ByteVector32, channel: ActorRef, state: ChannelState, data: ChannelData) extends CommandSuccess[CMD_GET_CHANNEL_INFO]
```

Reply:
Right, since `ActorRef`s are `Comparable`, this is better. Done in https://github.com/ACINQ/eclair/pull/2688/commits/61bce78654e0b9c3d32922a585d444e45bf70c7d